dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/CRUD/ConformanceTestsBase.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using VectorDataSpecificationTests.Support;
+
+namespace VectorDataSpecificationTests.CRUD;
+
+// TKey is a generic parameter because different connectors support different key types.
+public abstract class ConformanceTestsBase<TKey, TRecord>(VectorStoreFixture fixture) where TKey : notnull
+{
+    protected VectorStoreFixture Fixture { get; } = fixture;
+
+    protected virtual string GetUniqueCollectionName() => Guid.NewGuid().ToString();
```

Comment:
In some databases, creating collections is a heavy operation, so we should generally try to avoid creating a collection for each test unless it's necessary. This is why the filter tests have a single collection, and all tests reuse it. I think that should be possible for the new tests being introduced here.